src/lib/components/Swap/Settings/MaxSlippageSelect.tsx

```diff
@@ -1,9 +1,10 @@
-import { t, Trans } from '@lingui/macro'
+import { Trans } from '@lingui/macro'
```

Comment:
These changes aren't spec'ed out (@ppphhhiiilll), and you've changed the UI so that the button styles are now different. Could you back out all changes to the UI of this file, and just change the functional aspect?

I think we need to get a spec before we add the warnings that you've added here.